### ISSUE TITLE
Icon Request: .ejs extension

### ISSUE THREAD

**Issue by mert-celen** [robertohuertasm/vscode-icons#165]

**Comment by maqboolkhan** [comment]
:+1:

**Comment by robertohuertasm** [comment]
@mert-celen please, I don't mean to be rude but we have issue templates for something. Please try to stick to them as close as possible. Does it has some sort of official logo? Url of the extension... you know...
Thanks.

**Comment by iRoachie** [comment]
@robertohuertasm Is this sufficient? This is the icon from atom since there's no official ejs icon. 
<img width="23" alt="screen shot 2016-08-20 at 8 13 29 am" src="https://cloud.githubusercontent.com/assets/5962998/17831094/0f171ebe-66ae-11e6-8c3c-e7090a0c43f1.png">

**Comment by mert-celen** [comment]
Sorry for late reply, I didn't see any issue template sorry about that. Regarding the official logo, this is from google code repo of the ejs (https://storage.googleapis.com/google-code-archive/v2/code.google.com/embeddedjavascript/logo.png). This should give you an idea. Here's the official website of the ejs http://www.embeddedjs.com/

_Ps: I did actually write explanation in issue and send it that way, I seriously don't know what happened. Maybe it's gone when I pressed back button from browser after a accident click. Sorry about that._
